test/e2e/fixtures/KeyboardNavigation/GridFocus.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+
+const baselineProps = {
+  rows: [
+    {
+      id: 0,
+      brand: 'Nike',
+    },
+    {
+      id: 1,
+      brand: 'Adidas',
+    },
+    {
+      id: 2,
+      brand: 'Puma',
+    },
+  ],
+  columns: [{ field: 'brand', width: 100 }],
+};
+
+export default function BaseGridFocus() {
```

Comment:
I don't think that the file structure is flying. Material-UI X while host more components. I think that a prefix with the name of the component and a flat structure after would work a lot better. So 👍 for ` test/e2e/fixtures/DataGrid/KeyboardNavigationFocus.tsx`

Reply:
Yes, I was not convinced about the naming. I'll update it

Reply:
Updated